dbfit-java/informix/src/main/java/dbfit/environment/InformixBigDecimalTransformer.java

```diff
@@ -0,0 +1,20 @@
+package dbfit.environment;
+
+import java.sql.SQLException;
+
+import dbfit.util.TypeTransformer;
+
+public class InformixBigDecimalTransformer implements TypeTransformer {
+
+    @Override
+    public Object transform(Object o) throws SQLException {
+        if (o == null) {
+            return null;
+        }
+        if (!(o instanceof dbfit.util.NormalisedBigDecimal)) {
+            throw new UnsupportedOperationException("InformixBigDecimalSpecifier cannot work with " + o.getClass());
```

Comment:
```
The TypeTransformer's transform specifies that throws a SQLException should be thrown. But the new Informix transformer (and all of the existing xNormaliser classes thow an UnsupportedOperationException). This isn't quite right is it? If not can we move that out to a new issue instead of fixing it here?
```

In the particular case - maybe sticking to unchecked exceptions would be more appropriate.
https://docs.oracle.com/javase/tutorial/essential/exceptions/runtime.html